lib/mix/tasks/docs.ex

```diff
@@ -54,6 +54,14 @@ defmodule Mix.Tasks.Docs do
       directory in the output path. Its entries may be referenced in your docs
       under "assets/ASSET.EXTENSION"; defaults to no assets directory.
 
+    * `:before_closing_body_tag` - Literal HTML to be included just before the closing body tag (`</body>`)
+      Useful to inject custom assets, such as Javascript. 
+      Only works with the HTML formatter.
```

Comment:
@tmbb @josevalim Why? This should also work with EPUB documents, at the end, an EPUB file is just a bunch of `.xhtml` files.